src/AnalysisServices/AnalysisServices/help/New-AzAnalysisServicesServer.md

```diff
@@ -38,7 +38,10 @@ Creates a new Analysis Services server. (autogenerated)
 
 <!-- Aladdin Generated Example -->
 ```powershell
-New-AzAnalysisServicesServer -Administrator 'testuser1@contoso.com' -FirewallConfig <PsAzureAnalysisServicesFirewallConfig> -Location 'West-US' -Name 'testserver' -ResourceGroupName 'testresourcegroup' -Sku 'S1'
+$rule1 = New-AzAnalysisServicesFirewallRule -FirewallRuleName rule1 -RangeStart 0.0.0.0 -RangeEnd 255.255.255.255
+$rule2 = New-AzAnalysisServicesFirewallRule -FirewallRuleName rule2 -RangeStart 6.6.6.6 -RangeEnd 7.7.7.7
```

Comment:
This is  Aladdin generated example. I'm not sure whether will be override in future. Could you confirm @navba-MSFT

Reply:
AFAIK, The newly updated content should be retained.